.github/workflows/yarn_audit.yml

```diff
@@ -0,0 +1,24 @@
+# When the action is executed, it runs yarn audit command in all the paths that is mentioned in the input. The exit codes of the commands are compared and if it is greater than 7 (only high severity as of now), the action will try to fetch the open issues in the repo with the label provided in the input. The label is mandatory to prevent from creating duplicate issues. If there are no open issues with the given label in open state, the action will try to create a Github Issue with the details provided in the input.
+
+name: Full Clients Audit
+
+on:
+  push:
+    branches: [development, development_2.6]
```

Comment:
I think we should remove ```development_2.6``` as it's only a temporary branch and will be deleted when merged to the ```development``` branch

Reply:
Same branch is used in all workflows. We can update all workflows once it's merged.